src/lwc/bdiFieldMappingModal/bdiFieldMappingModal.js

```diff
@@ -205,9 +206,11 @@ export default class bdiFieldMappingModal extends LightningElement {
         try {
             this.hasSourceFieldErrors = false;
             this.hasTargetFieldErrors = false;
+            this.collectMappedTargetMappings(event.fieldMappings);
             this.objectMapping = event.objectMapping;
-
-            this.collectMappedDataImportFields(event.fieldMappings);
+            this.diFieldDescribes = event.diFieldDescribes;
+            this.mappedDiFieldDescribes = event.mappedDiFieldDescribes;
```

Comment:
This is very minor feedback, but prefer "mappedDIFieldDescribes" or "mappedDataImportFieldDescribes"